code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -48,6 +48,7 @@
 			chamber_round()
 			A.update_icon()
 			update_icon()
+			user.say(pick("I'M RELOADING!!!", "NEW MAG IN!!!", "CHANGING MAG!!!", "COVER ME I'M RELOADING!!!", "OUT OF AMMO!!!", "RELOADING!!!"))

```

Comment:
There should be a comma in between COVER ME and I'M RELOADING
COVER ME, I'M RELOADING